README.md

```diff
@@ -2,9 +2,16 @@
 
 This repository contains the scripts and Terraform configurations required to set up a Concourse pipeline for building Bosh releases. Releases each have their own repository and are stored in an S3 bucket when they are built.
 
+
+## Automatic build
+
+All releases that have a [pipeline defined](https://github.com/alphagov/paas-release-ci/blob/master/scripts/build-boshrelease-pipelines.sh) in this repository, as well as releases defined in PRs raised from a branch of this repository, are automatically built by `build concourse`. Releases on master branch get final release builds, whereas releases from PRs get development release build.
```

Comment:
I think the file link can be relative like `[pipeline defined](scripts/build-boshrelease-pipelines.sh)` so that it doesn't break if we rename the repo (and I think it will respect the current branch you're viewing).

ref: https://help.github.com/articles/about-readmes/#relative-links-and-image-paths-in-readme-files